src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2108,8 +2108,8 @@ Signing is only possible with addresses of the type 'legacy'.</source>
         <translation>訊息(&amp;M):</translation>
     </message>
     <message>
-        <source>An optional message to attach to the payment request, which will be displayed when the request is opened. Note: The message will not be sent with the payment over the Bitcoin network.</source>
-        <translation>附加在付款要求中的訊息，可以不填，打開要求內容時會顯示。注意: 這個訊息不會隨著付款送到 Bitcoin 網路上。</translation>
+        <source>An optional message to attach to the payment request, which will be displayed when the request is opened. Note: The message will not be sent with the payment over the Dogecoin network.</source>
+        <translation>附加在付款要求中的訊息，可以不填，打開要求內容時會顯示。注意: 這個訊息不會隨著付款送到 Dogecoin 網路上。</translation>
```

Comment:
Translate "dogecoin" to "狗狗幣" and retouch the sentence.
```suggestion
        <translation>(選填)附加在付款要求中的訊息，打開要求內容時會顯示。注意: 這個訊息不會隨著付款送到狗狗幣網路上。</translation>
```